Application/Transfer/Import/EVMS/Service.php

```diff
@@ -0,0 +1,914 @@
+                                        TblCommonInformation::VALUE_IS_ASSISTANCE_NULL,
+                                        '',
+                                        ''
+                                    );
+
+                                    // Address
+                                    $cityName = trim($Document->getValue($Document->getCell($Location['Ort'], $RunY)));
+                                    $cityDistrict = trim($Document->getValue($Document->getCell($Location['OT'], $RunY)));
+                                    list($streetName, $streetNumber) = $importService->splitStreet('Adresse', $RunY);
+                                    $streetNumber = str_replace(' ', '', $streetNumber);
+
+                                    if ($streetName !== '' && $streetNumber !== '' && $cityCode && $cityName) {
+                                        Address::useService()->insertAddressToPerson(
+                                            $tblPerson, $streetName, $streetNumber, $cityCode, $cityName, $cityDistrict, ''
+                                        );
+                                    } else {
+                                        $error[] = 'Zeile: ' . ($RunY + 1) . ' Die Adresse ist nicht vollständig.';
+                                    }
+
+                                    $importService->insertBusinessMail($tblPerson, 'Email', $RunY);
+                                    $importService->insertPrivatePhone($tblPerson, 'Tel.:', $RunY);
+                                }
+                            }
+                        } else {
+                            $error[] = 'Zeile: ' . ($RunY + 1) . ' Die Person wurde nicht angelegt, da sie keinen Namen und Vornamen hat.';
+                        }
+                    }
+
+                    return
+                        new Success('Es wurden ' . $countStaff . ' Mitarbeiter erfolgreich angelegt.') .
+                        ($countStaffExists > 0 ?
+                            new Warning($countStaffExists . ' Mitarbeiter exisistieren bereits.') : '')
+                        . new Layout(new LayoutGroup(new LayoutRow(new LayoutColumn(
+                            new Panel(
+                                'Fehler',
+                                $error,
+                                Panel::PANEL_TYPE_DANGER
+                            )
+                        ))));
+
+                } else {
+                    return new Warning(json_encode($Location)) . new Danger(
+                            "File konnte nicht importiert werden, da nicht alle erforderlichen Spalten gefunden wurden");
+                }
+            }
+        }
+
+        return new Danger('File nicht gefunden');
+    }
+
+    /**
+     * @param IFormInterface|null $Form
+     * @param UploadedFile $File
+     *
+     * @return IFormInterface|Danger|string
+     */
+    public function createCompaniesFromFile(IFormInterface $Form = null, UploadedFile $File = null)
```

Comment:
Codacy found an issue: [The method createCompaniesFromFile() has 200 lines of code. Current threshold is set to 100. Avoid really long methods.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)